### DISCUSSION TITLE
Is there some RSS-like API to auto get news in the daily.dev?

### DISCUSSION THREAD

**Discussion by shaokeyibb** [dailydotdev/daily#785] [Feature Requests]
Just want to build a news rebot to my social group

**Comment by idoshamun** [comment]
Hi @shaokeyibb,

Unfortunately there is not such an api. With that said, we do have plans to support social groups within our platform

**Reply by shaokeyibb** [reply]
Okay, thanks!

**Reply by naquad** [reply]
Am I getting it right that daily.dev wants to replace the user's RSS reader?

I like the functionality (filtering, summaries, ...), but I'm very far from being ready to abandon Tiny Tiny RSS setup I have, especially given the number of private feeds I'm subscribed to.

It would be great to be able to have a personalized RSS with all the filters set.

**Comment by raiftekin** [comment]
Sub.

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
